test/compile/data/bin.test.ts

```diff
@@ -50,18 +50,60 @@ describe('compile/data/bin', function() {
         assert.deepEqual(transform[0], {
           type: 'extent',
           field: 'Acceleration',
-          signal: 'Acceleration_extent'
+          signal: "bin_maxbins_10_Acceleration_extent"
         });
         assert.deepEqual(transform[1], {
           type: 'bin',
           field: 'Acceleration',
-          as: ['bin_Acceleration_start', 'bin_Acceleration_end'],
+          as: ["bin_maxbins_10_Acceleration_start", "bin_maxbins_10_Acceleration_end"],
           maxbins: 10,
-          signal: "Acceleration_bins",
-          extent: {signal: 'Acceleration_extent'}
+          signal: "bin_maxbins_10_Acceleration_bins",
+          extent: {signal: 'bin_maxbins_10_Acceleration_extent'}
         });
       });
     });
+    describe('should bin the same field', () => {
```

Comment:
why using `describe` here?  This definitely looks like an `it`.

Also please update description to be clearer (like `it('should ... when two channels encode the same binned field', `)